tower-http/Cargo.toml

```diff
@@ -34,15 +34,13 @@ tower = "0.4"
 [features]
 default = []
 full = [
-    "br",
-    "decompression",
-    "deflate",
-    "gzip",
+    "decompression-full",
 ]
-br = ["async-compression/brotli"]
 decompression = ["bytes", "tokio-util"]
-deflate = ["async-compression/zlib"]
-gzip = ["async-compression/gzip"]
+decompression-full = ["decompression-br", "decompression-deflate", "decompression-gzip"]
```

Comment:
I actually think this is a nice approach to let users easy enable all the things, while also letting some customize stuff. I'll do the same thing on `Compression`. Should allow us to move forward with this one.